src/systems/systems/src/hydraulic/mod.rs

```diff
@@ -193,15 +195,48 @@ pub struct PowerTransferUnit {
 
     is_in_continuous_mode: bool,
     is_rotating_after_delay: DelayedTrueLogicGate,
+
+    activation_delta_pressure: Pressure,
+    desactivation_delta_pressure: Pressure,
+    shot_to_shot_activation_coefficient: f64,
+    shot_to_shot_desactivation_coefficient: f64,
+
+    duration_since_active: Duration,
+    speed_captured_at_active_duration: AngularVelocity,
+    bark_strength: u8,
+    has_stopped_since_last_write: bool,
+
+    efficiency: f64,
 }
 impl PowerTransferUnit {
-    const ACTIVATION_DELTA_PRESSURE_PSI: f64 = 500.;
-    const DEACTIVATION_DELTA_PRESSURE_PSI: f64 = 35.;
+    // Randomisation parameters
+    // As ptu wear parameters are non linear, for now we simulate two normal distributions:
+    // -Nominal distribution which is the PTU you'll find in wide majority of planes
+    // -Worn out distribution, which is the PTU which is still acceptable but has degraded behaviour
+    const MEAN_ACTIVATION_DELTA_PRESSURE_PSI: f64 = 500.;
+    const STD_DEV_ACTIVATION_DELTA_PRESSURE_PSI: f64 = 5.;
+
+    // Ratio of worn PTU : 0.1 means 10% of cases we get a worn out ptu
+    const WORN_PTU_CASE_PROBABILITY: f64 = 0.15;
+
+    const WORN_MEAN_DEACTIVATION_DELTA_PRESSURE_PSI: f64 = 20.;
+    const WORN_STD_DEV_DEACTIVATION_DELTA_PRESSURE_PSI: f64 = 5.;
+    const WORN_MIN_DEACTIVATION_DELTA_PRESSURE_PSI: f64 = 5.;
+    const WORN_MAX_DEACTIVATION_DELTA_PRESSURE_PSI: f64 = 30.;
+
+    const NOMINAL_MEAN_DEACTIVATION_DELTA_PRESSURE_PSI: f64 = 90.;
+    const NOMINAL_STD_DEV_DEACTIVATION_DELTA_PRESSURE_PSI: f64 = 15.;
+    const NOMINAL_MIN_DEACTIVATION_DELTA_PRESSURE_PSI: f64 = 5.;
+    const NOMINAL_MAX_DEACTIVATION_DELTA_PRESSURE_PSI: f64 = 180.;
+
```

Comment:
I might just pass those parameters as constructor parameters maybe, so the random choices go at A320 factory level? Practically it won't change much for us though as we'll probably have only one PTU type simulated for a long long time though, but would make sense for coherency indeed.

For testability it's far from easy, couple of remarks:
- First, note that those parameters are not meant to represent any kind of failed state, so any random state shall pass tests
- If running tests with only one set of chosen parameters instead of random OR if using always same seed, it will never ensure that any random state can pass tests. I'd say you even have statistically more chances to catch a test fail letting tests running on random values than using same seed so you just only always test the same parameters.
- For testability I don't see any magic solution. The best I could imagine would be to run tests for one worst case ptu and one best case ptu. But again you'd have to run tests twice... and if you have other components you'd have to run everything with all combinations of best and worst parts.... See the factorial nightmare :D
- Maybe adding one dedicated test wisely chosen for best part and worst part is the best realistic and practical solution?